docs/user-guide/ze-cics.md

```diff
@@ -0,0 +1,267 @@
+
+- [Features](#features)
+- [System requirements](#system-requirements)
+- [Installing](#installing)
+    - [Installing from Visual Studio Code](#installing-from-visual-studio-code)
+    - [Installing from a VSIX file](#installing-from-a-vsix-file)
+- [Getting started](#getting-started)
+    - [Creating profile](#creating-profile)
+    - [Updating profile](#updating-profile)
+    - [Hiding profiles](#hiding-profiles)
+    - [Deleting profiles](#deleting-profiles)
+- [CICS resources](#cics-resources)
+    - [Showing and filtering resources in a region](#showing-and-filtering-resources-in-a-region)
+    - [Showing and filtering resources in a plex](#showing-and-filtering-resources-in-a-plex)
+    - [Showing and filtering resources in an 'All' resource tree](#showing-and-filtering-resources-in-an-all-resource-tree)
+    - [Showing attributes](#showing-attributes)
+    - [Enabling and disabling](#enabling-and-disabling)
+    - [New copy and phase in](#new-copy-and-phase-in)
+    - [Opening and closing local files](#opening-and-closing-local-files)
+- [Untrusted TLS certificates](#untrusted-tls-certificates)
+- [Usage tips](#usage-tips)
+- [Providing feedback or contributing](#providing-feedback-or-contributing)
+    - [Checking the source of an error](#checking-the-source-of-an-error)
+    - [Filing an issue](#filing-an-issue)
+- [Uninstalling](#uninstalling)
+
+## Features
+
+- **Load profiles** directly from a locally installed Zowe instance. 
+- Create new Zowe CICS profiles and connect to them.
+- **Update** session details, and **delete** profiles by using the user-friendly interface.
+- **Work with multiple regions** that contain programs, local transactions, and local files within a plex in a comprehensible tree-like format.
+- Perform actions such as **Enable**, **Disable**, **New Copy**, and **Phase In** directly from the UI.
+- Perform additional actions on local files including **Open** and **Close** directly from the UI.
+- View and search attributes of resources and regions by right-clicking and using the dynamic filtering feature.
+- **Apply multiple filters** to regions, programs, local transactions, and local files.
+- View and interact with all resources under a plex.
+
+## System requirements
+
+Install [Visual Studio Code](https://code.visualstudio.com/download) on your PC.
+
+## Installing
+
+You can install or update the extension from Visual Studio Code Extensions or from a VSIX file.
+
+- [Installing from Visual Studio Code Extensions](#installing-from-visual-studio-code-extensions)
+- [Installing from a VSIX file](#installing-from-a-vsix-file)
+
+### Installing from Visual Studio Code Extensions
+
+1. Navigate to **Extensions** tab of your VS Code application.
+
+2. Search for `Zowe Explorer for IBM CICS` and click it.
+
+3. Click **Install** at the top of the page.
+
+   If you do not have Zowe Explorer installed, it will automatically install it for you as part of the installation.
+
+### Installing from a VSIX file
+
```

Comment:
For this method, Zowe Explorer is a required prerequisite, so suggest adding a sentence to ask users to check that before performing the steps below. Something like this: 
`Before you perform the following steps, ensure that you have installed Zowe Explorer which is a required dependency. For more information, see [Installing Zowe Explorer](https://docs.zowe.org/stable/user-guide/ze-install#installing).`

Reply:
done